src/CattyTests/DataModel/UserDataContainerTest.swift

```diff
@@ -409,19 +410,19 @@ final class UserDataContainerTest: XCTestCase {
         let userVariable3 = UserVariable(name: "testName3")
         let userVariable4 = UserVariable(name: "testName4")
 
-        var result = container.addObjectVariable(userVariable1, for: objectA)
+        var result = objectA.userData.addVariable(userVariable1)
         XCTAssertTrue(result)
 
-        result = container.addObjectVariable(userVariable2, for: objectB)
+        result = objectB.userData.addVariable(userVariable2)
         XCTAssertTrue(result)
 
-        container.programVariableList.add(userVariable3 as Any)
+        container.addVariable(userVariable3)
 
-        XCTAssertTrue(container.getUserVariableNamed(userVariable1.name, for: objectA)?.isEqual(userVariable1) == true)
-        XCTAssertTrue(container.getUserVariableNamed(userVariable1.name, for: objectB) == nil)
-        XCTAssertTrue(container.getUserVariableNamed(userVariable3.name, for: objectA)?.isEqual(userVariable3) == true)
-        XCTAssertTrue(container.getUserVariableNamed(userVariable3.name, for: objectB)?.isEqual(userVariable3) == true)
-        XCTAssertTrue(container.getUserVariableNamed(userVariable4.name, for: objectA) == nil)
+        XCTAssertTrue(objectA.userData.getUserVariable(withName: userVariable1.name)?.isEqual(userVariable1) == true)
+        XCTAssertTrue(objectB.userData.getUserVariable(withName: userVariable1.name) == nil)
+        XCTAssertTrue(container.getUserVariable(withName: userVariable3.name)?.isEqual(userVariable3) == true)
```

Comment:
This line is duplicated but there is an assert missing for `objectB` and `userVariable2`.